### ISSUE TITLE
BUG: HoC does not work with mutations

### ISSUE THREAD

**Issue by geekflyer** [tomitrescak/apollo-mobx#1]
If I try to use the `grapqhl` hoc with a mutation my app crashes upon initialization with
`Uncaught (in promise) Error: Must contain a query definition.`

For example:

```tsx
const createOrgGroupMutation = gql`
    mutation createOrgGroup ($name: String!) {
        createOrgGroup(name: $name) {
            id
        }
    }
`;

export default graphql(createOrgGroupMutation)(CreateOrgGroupDialog);

```

If I use the graphql hoc from react-apollo this error does not occur.

**Comment by tomitrescak** [comment]
Yeah, that is not implemented. I use mutations differently using the "mutation" construct from this package. As I have usually 4-5 mutations per component that seemed to be a better option. Once I'll have a bit of time I'll look into this.

**Comment by geekflyer** [comment]
@tomitrescak which mutation construct are you referring to? could you perhaps create an example? thanks
